x/incentives/client/cli/query.go

```diff
@@ -234,6 +235,47 @@ $ %s query incentives active-gauges
 	return cmd
 }
 
+// GetCmdActiveGaugesPerDenom returns active gauges for specified denom
+func GetCmdActiveGaugesPerDenom() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "active-gauges-per-denom [denom]",
+		Short: "Query active gauges per denom",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Query active gauges.
+
+Example:
+$ %s query incentives active-gauges-per-denom [denom]
+`,
+				version.AppName,
+			),
+		),
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := types.NewQueryClient(clientCtx)
+			pageReq, err := client.ReadPageRequest(cmd.Flags())
+			if err != nil {
+				return err
+			}
```

Comment:
Just flagging this, we should make a helper method somewhere in our repo that we can use to do all of these in one line, to reduce our boilerplate. We should probably make a utils folder in our codebase somewhere, and make an issue in the SDK.

Gotta eliminate this boilerplate from the SDK :/

(Doesn't block this PR at all)

Reply:
Yep, totally agreed, should we put this issue as a backlog?

Reply:
Yes please. Really, we should make an issue for it on the SDK. If you have bandwidth to do so, would be very appreciated.